test/config/stack_config.json

```diff
@@ -64,6 +64,23 @@
       }
     }
   },
+  "docker_local_run":{
+    "rackhd_host": "localhost",
+    "type": "linux",
+    "ucsm_ip" : "172.31.128.252",
+    "ucs_service_uri" : "https://localhost:7080",
+    "ucsm_config_file" : "./tests/ucs/ucsm_config.xml",
+    "install-config": {
+      "ports": {
+        "mongo_port": 27017,
+        "http": 9090,
+        "https": 8443,
+        "amqp_ssl": 5672,
+        "amqp_nossl": 5671,
+        "ssh": 22
```

Comment:
"FIT and CIT should run with only the access of the API , for example:
http://localhost:9090, amqp://guest:guest@localhost:5672" -- first, there is only FIT, no CIT. We've been trying to collapse the two together, but honestly spend maybe a 3rd of our time dealing with unify the testing.

More importantly, the FIT framework is meant to be able to DO all the environment mapping for our testing. Continuous testing like PR gates, deployment spreads, per-merge, extended and so on. Being able to go into the box under any deployment to allow tests to execute anywhere as well as do things like inject errors and so on. (virtualenv isn't part of this, to my awareness). I TOTALLY AGREE ABOUT RUNNING SSH IN DOCKER, but instead of removing a bunch of our capabilities, look at providing primitives to do "on box" operations that do ssh when ssh is there, and "docker exec ..." for (example) docker. 

At the very least in this case, 'ssh' is the wrong name for the field. It should be something like 'image_controller_ssh_port' (along with 'image_controller_host' (and auth data) -- other examples might be the v-center host for deploying OVAs). I.E 'ssh' is used throughout the testing environment to mean 'how to ssh into the RackHD-under-test"